### DISCUSSION TITLE
Creating reports for Projects v2

### DISCUSSION THREAD

**Discussion by dpal** [community/community#40569] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hello,

The documentation here: https://docs.github.com/en/issues/planning-and-tracking-with-projects/viewing-insights-from-your-project/configuring-charts clearly describes how to customize your report.

See step 6: To change the field used for your chart's X-axis, select the X-axis dropdown and click the field you want to use.
In the dropdown, there is an option to select Time.

There is no such option in the Project that I am using. Is there a particular configuration to enable it?

Thank you!

**Comment by ossanna16** [comment]
Hi there @dpal and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by chiedo** [comment]
Hey @dpal!

Have you confirmed that you have one of the plans that have the "Time" option enabled? (either the Team or Enterprise Cloud plan)

**Reply by dpal** [reply]
Hello @chiedo, thank you for your response. It was not clear that this option is only available for paid accounts.

**Reply by chiedo** [reply]
I'll share that feedback with my colleagues!
